client/components/widget/data_viz/gviz/gviz-directive_test.js

```diff
@@ -84,7 +84,10 @@ describe('gvizDirective', function() {
                 /.*pk-cond-(\S+)/, '$1');
 
             var result = {};
-            result.pass = (parentState == expected);
+            var classNames = parentState.split(' ');
```

Comment:
Did something get out of sync, or is this a side effect of splitting branches? This shouldn't be part of this PR, merges should IMHO be separate.